CONTRIBUTING.md

```diff
@@ -0,0 +1,191 @@
+
+
+## Submit it for review
+
+Now that you have your contribution tested, your changes committed to your local git repository with a conforming commit message, you can push your changes to your personal git repository:
+
+
+```
+    > git push origin ps-8.0-8675
+```
+
+
+Next, submit it to us for review.  This will be done entirely through the GitHub Web User Interface.  Logon to your git repository via the web, navigate to your forked percona-server repository, find your recently pushed branch, click **Compare & pull request**.  Before you confirm, take this opportunity to review your diff and make any changes.  Once you are happy with your changes and commit message, confirm by clicking **Create pull request**.
+
+Once you have confirmation of your pull request, it is a good idea to copy the link and paste it into the JIRA issue as a new comment.  This makes it easier to find it later on.
+
+From here, one of our engineers will adopt your contribution.  It will begin with a thorough code review for functionality, behavioral changes, coding style, security, test coverage, etc…  Chances are very good that we will ask for changes during the review process.  When you make additional changes, please do not re-commit and force push your git branch.  We prefer that you just make additional commits to your original work and push to your working branch within your personal git repo.  The pull request will be automatically updated with your additional commits.  Usually just replying to or adding a new comment within the PR to let us know of your change is enough.
+
+Once our Engineers are satisfied with the state of the work, we will run the change through our internal CI/CD system and post a comment or link to the results to the pull request and/or JIRA issue (you will not be able to view the results).  If we notice any unexpected failures in the CI/CD test jobs, it will be communicated to you within the pull request what tests failed and you will be asked to correct the failures.
+
+Now we are ready to merge, but wait, there is one more thing.  Remember those additional corrective commits that you made to your working branch?  Well, you will now be asked to “squash” all of those into a single commit and force push your branch.  If your fork allows changes to be made by the maintainers, one of our engineers can do this for you as well as format the commit message.
+
+And that is it!  We will select a release target for your work and make sure that it is all properly documented and listed within our release notes.
+
+
+## What if I was working on something in a lower major series?
```

Comment:
Is this section still relevant with the linear 8.0 branch?

Reply:
That's a really good catch.  I need to rethink this part.